### PR TITLE
Update docs to include files.uploadV2

### PR DESCRIPTION
**Pull request by srajiang**
###  Summary

Adds docs for files.uploadV2 wrapper method. 

### Requirements (place an `x` in each `[ ]`)

* [x] I've read and understood the [Contributing Guidelines](https://github.com/slackapi/node-slack-sdk/blob/master/.github/contributing.md) and have done my best effort to follow them.
* [x] I've read and agree to the [Code of Conduct](https://slackhq.github.io/code-of-conduct).

### CODE REVIEW

File: docs/_packages/web_api.md [hunk 1]
```diff
@@ -448,15 +448,77 @@ retrying the API call. If you'd like to opt out of that behavior, set the `rejec
 ---
 
 ### Upload a file
+As of @slack/web-api v2.8.0, we have introduced a modified way to upload files.
```
Comment (seratch) [inline docs/_packages/web_api.md:451]:
```suggestion
As of @slack/web-api v6.8.0, we have introduced a modified way to upload files. 
```

**Review by seratch — APPROVED** [review]
@srajiang Once the version error is fixed, all look good to me

File: docs/_packages/web_api.md (hunk 1 above)
Comment (srajiang) [reply docs/_packages/web_api.md]: Ooh, thanks for the catch! I've fixed it.

**Comment by seratch** [comment]
Thanks, this is now perfect ✅ Let's merge this PR once we ship the version!

**Comment by hello-ashleyintech** [comment]
6.8.0 has been released: https://github.com/slackapi/node-slack-sdk/releases/tag/%40slack%2Fweb-api%406.8.0 will merge this PR in now!

**Comment by hello-ashleyintech** [comment]
actually, it looks like I don't have admin rights and can't get past the 12.x check hanging 😬 will get someone with admin rights to merge this!
Screen Shot 2022-11-08 at 3 15 58 PM

**Pull request merged**